edx-solutions/edx-platform
# Bump openedx-completion-aggregator

Description
------------
This PR bumps `openedx-completion-aggregator` to fix a serialisation error.

JIRA tickets
------------
MCKIN-14043
MCKIN-14046
MCKIN-14177
MCKIN-14178

Dependencies
----------------
https://github.com/open-craft/openedx-completion-aggregator/pull/68

Testing instructions
--------------------

1. Deploy this to integration or another environment that makes use of Celery tasks
1. Enroll a user in a course
1. Uneroll a user from a course
1. Create a group of students in Group Work
1. Assign students to a Program
1. Register for course access
1. Ensure no failures or `Internal Server Error` messages were generated

Author notes and concerns
----------------------------

1. **Merge deadline**: ASAP
1. There's no way to test this in a devstack yet

Reviewers
-----------
